api/src/main/java/org/openmrs/Encounter.java

```diff
@@ -486,6 +489,26 @@ public void setDiagnoses(Set<Diagnosis> diagnoses) {
 		this.diagnoses = diagnoses;
 	}
 	
+	/**
+	 * Basic property getter for conditions
+	 * 
+	 * @return conditions - set of conditions
+	 * @since 2.4.0
+	 */
+	public Set<Condition> getConditions() {
+		return conditions;
+	}
+
+	/**
+	 * Basic property setter for conditions
+	 *  
+	 * @param conditions - set of conditions
+	 * @since 2.4.0
+	 */
+	public void setConditions(Set<Condition> conditions) {
+		this.conditions = conditions;
+	}
+
```

Comment:
Then I would expect test showing that conditions can be set and gotten from an encounter.

Reply:
Added test cases for this